README.md

```diff
@@ -1,2 +1,47 @@
-Galaxy Mentor Network 
+Galaxy Mentor Network
 =====================
+
+# How can I generate the website locally?
+
+### 1. Install Gridsome CLI tool if you don't have
```

Comment:
It may nice to have instructions as in the Hub: https://github.com/galaxyproject/galaxy-hub/#getting-started